Lexer FSM/main.py

```diff
@@ -6,39 +6,74 @@
 from Keywords import validate_keyword, keywords
 
 def main():
+    with open('./input.txt', 'r') as file:
```

Comment:
This is a good implementation, reading the file from an input.txt since that's where the test cases are coming from